doc-content/enterprise-only/openshift/imagestreams-file-install-amq-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='imagestreams-file-install-amq-proc']
+= Ensuring the availability of AMQ scaledown controller image streams for a high-availability deployment
+
+If you want to deploy a {PRODUCT} high-availability authoring environment, you must ensure that image streams for AMQ scaledown controller are available.
+
+.Prerequisites
+* You ensured the availability of image streams and the image registry for {PRODUCT} according to instructions in <<imagestreams-file-install-proc>>.
+
+.Procedure
+
+If you want to deploy a high-availability {CENTRAL} (this functionality is a technology preview), complete the following additional steps:
```

Comment:
I don't understand this bit and why it's placed here. This should be moved to the intro segment after the procedure title, but even then, what additional steps? Do you mean that if users are deploying HA Central, then the steps in this procedure are also required? If so, I'd just say that.

This is what I'd expect the intro to be, based on my assumptions:
```
= Ensuring ...

If you want to deploy a {PRODUCT} high-availability authoring environment, or if you want to deploy a high-availability {CENTRAL}, you must ensure that image streams for AMQ scaledown controller are available.

NOTE: <Repeat tech preview boiler plate about HA Central. But looking at what you put elsewhere, I still say that "In the current version, " is vague. Suggest "In {PRODUCT} {PRODUCT_VERSION}, ">
```